.github/ISSUE_TEMPLATE/story.md

```diff
@@ -38,13 +38,10 @@ As a _, so that _, I need _.
 - [ ] Code quality tasks completed
   - [ ] Code refactored for clarity and no design/technical debt
   - [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies
-  - [ ] Code is reviewed or developed by pair; 1 approval is needed but consider requiring an outside-the-pair reviewer
-  - [ ] Code quality checks passed
-- [ ] Security & Privacy tasks completed
-  - [ ] Security & privacy gates passed
 - [ ] Testing tasks completed
   - [ ] Load tests passed
-  - [ ] Unit test coverage of our code >= 90%
```

Comment:
Is unit testing now implied/as necessary?

Reply:
I am also curious

Reply:
This is enforced by `jacocoTestCoverageVerification` and it's a PR-level requirement. It makes sense to remove it as we wouldn't be able to merge a PR if we had less test coverage. At least in the TI repo

Reply:
That's good to know. Looks like SFTP has a threshold.